spec/models/rating/data_spec.rb

```diff
@@ -21,7 +21,14 @@
     end
 
     it 'returns the estimate for a resource' do
-      expect(result[:estimate].to_s).to eq '42.5000000000000000012000000505'
+      case ENV.fetch('DB')
+      when 'mysql'
+        expect(result[:estimate]).to eq(BigDecimal('42.5000000000000000012000000505'))
```

Comment:
@wbotelhos So the other two are different precisions, but this is actually a meaningfully different value.  Why is Postgres computing a different value here?  Precision of the intermediate values?

Reply:
Maybe it's the precision along with all decimal fields which result in the `estimate` field. I think the precision is not our issue, since how keeps it is not the application. So, let's just use round and go forward? We can add an observation in the README.md about different precisions between MySQL and Postgres. WDYT?